user-creation-testing/src/main/java/no/unit/nva/useraccessservice/userceation/testing/cristin/MockPersonRegistry.java

```diff
@@ -69,6 +69,14 @@ public CristinPerson getPerson(String nin) {
         return people.get(nin);
     }
 
+    public String personWithoutAffiliations() {
```

Comment:
personWithoutAffiliations method calls method createPersonWithAffiliations?

Reply:
or does that null value in parameter mean without?

Reply:
Yes. I agree it is confusing naming. I will fix it.